content/en/community.md

```diff
@@ -3,28 +3,77 @@ title: Community
 sidebar: false
 ---
 
-NumPy offers a number of communication channels available to individuals interested in using and contributing to the project.
 
-We welcome contributions and contributors of all kinds - whether they come as contributions to code, participation in the community, opening issues and pointing out bugs, or simply sharing your work with others.
+# Community
 
-### Join the NumPy community
+NumPy is an open source project developed by a very diverse group of [contributors](https://numpy.org/team.md).
 
-If you’re interested in joining the NumPy community (yay!) we recommend checking out the [NumPy Contributing guide](https://www.numpy.org/devdocs/dev/index.html). This contains information about ways to get involved, the skills that are useful for each project area or activity, how to get started working on code, documentation or the website, and other ways that you can become a part of the NumPy community.
+We offer several communication channels to learn, share your knowledge and connect with others within the NumPy community.
 
-### Participate online
+The NumPy leadership has made a strong commitment to creating an open, inclusive, and positive community. Please read the [NumPy Code of Conduct](https://www.numpy.org/code-of-conduct.md) for guidance on how to interact with others in a way that makes the community thrive.
 
-NumPy also has a number of online communication channels to help keep in touch.
+## Online Communities and Social Resources
 
-As you interact with others in the NumPy community, remember that we have a strong commitment to being an open, inclusive, and positive community. Please read the [NumPy Code of Conduct](https://www.numpy.org/devdocs/dev/conduct/code_of_conduct.html) for guidance on how to interact with others in a way that makes the community thrive.
+### [NumPy mailing list](https://mail.python.org/mailman/listinfo/numpy-discussion)
```

Comment:
The rendering as section headers looks a little odd. Content of the sections makes sense, however I'd rather use links in a more obvious way as part of a sentence or start of a list item.

Reply:
I didn't find a clearly better way, without going to a button solution like https://jupyter.org/community

Reply:
@rgommers The primary purpose of my PR was to upload the content. Good job on improving the formatting! 
Not sure about the buttons. What do you think about placing active icons before each header?

Reply:
> What do you think about placing active icons before each header?

Something like that could be nice. I'd like to postpone such changes till after we get the first version of the site up though - visual improvements like this are time consuming and aren't blockers.